.env.example

```diff
@@ -1,14 +1,26 @@
 # BOT
 SMF_BOT_BACKEND_URL="http://localhost:5555"
 SMF_BOT_DRIP_AMOUNT=10
+
+# Log in to the bots matrix account. Click on the name in the top left corner, then "Settings".
+# Click the "Help & About" tab (left side of the dialog).
+# Scroll to the bottom and click on <click to reveal> part of Access Token.
+# Once you logout - access token invalidates, so keep somewhere bot account logged in
 SMF_BOT_MATRIX_ACCESS_TOKEN="NotValidExampleToken"
-SMF_BOT_MATRIX_BOT_USER_ID="@NotValidExampleAccount:matrix.org"
+
+# Make sure to create an account within matrix.parity.io. You can use yourname+matrixbot@parity.io
+SMF_BOT_MATRIX_BOT_USER_ID="@NotValidExampleAccount:matrix.parity.io"
 SMF_BOT_NETWORK_DECIMALS=12
 SMF_BOT_NETWORK_UNIT="UNIT"
 SMF_BOT_FAUCET_IGNORE_LIST=""
```

Comment:
Add a comment on what this is used for

Reply:
this is explained in env.server.config.yml, here I think the explanation matters as Guide how to retrieve things that might be not clear, not explaining what it is